### PR TITLE
feat: Add version to the $ coder header in CLI

### PR DESCRIPTION
**Pull request by johnhorsema**

### CODE REVIEW

**Comment by johnhorsema** [comment]
I have read the CLA Document and I hereby sign the CLA

File: cli/root.go [hunk 1]
```diff
@@ -108,12 +108,13 @@ func AGPL() []*cobra.Command {
 }
 
 func Root(subcommands []*cobra.Command) *cobra.Command {
+	fmtLong := `Coder v%s — A tool for provisioning self-hosted development environments with Terraform.
```
Comment (ammario) [inline cli/root.go:111]:
```suggestion
	fmtLong := `Coder %s — A tool for provisioning self-hosted development environments with Terraform.
```

Remove the duplicate `v`

**Review by ammario — CHANGES_REQUESTED** [review]
Thank you for the contribution!

File: cli/root.go (hunk 1 above)
Comment (johnhorsema) [reply cli/root.go]: Removed in the latest commit [`84be5c7`](https://github.com/coder/coder/pull/4709/commits/84be5c752226ae6849bf1a59166801878a13a662)

**Review by ammario — APPROVED** [review]

**Pull request merged**